integrated_channels/degreed/client.py

```diff
@@ -42,6 +42,9 @@ def __init__(self, enterprise_configuration):
         self.global_degreed_config = apps.get_model('degreed', 'DegreedGlobalConfiguration').current()
         self.session = None
         self.expires_at = None
+        self.IntegratedChannelAPIRequestLogs = apps.get_model(
+            "integrated_channel", "IntegratedChannelAPIRequestLogs"
+        )
```

Comment:
Are we intentionally setting this here? 🤔

Reply:
@hamzawaleed01 it was redundant, removed it.